symbols/amenity/parcel_locker.svg

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!-- Generator: Adobe Illustrator 23.0.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+
+<svg
+   version="1.1"
+   id="Warstwa_1"
+   x="0px"
+   y="0px"
+   viewBox="0 0 14 14"
+   xml:space="preserve"
+   sodipodi:docname="parcel_locker.svg"
+   width="14"
+   height="14"
+   inkscape:version="1.1.1 (3bf5ae0d25, 2021-09-20)"
+   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
+   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
+   xmlns="http://www.w3.org/2000/svg"
+   xmlns:svg="http://www.w3.org/2000/svg"
+   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+   xmlns:cc="http://creativecommons.org/ns#"
+   xmlns:dc="http://purl.org/dc/elements/1.1/"><defs
+   id="defs69" /><sodipodi:namedview
+   id="namedview67"
+   pagecolor="#ffffff"
+   bordercolor="#666666"
+   borderopacity="1.0"
+   inkscape:pageshadow="2"
+   inkscape:pageopacity="0.0"
+   inkscape:pagecheckerboard="0"
+   showgrid="true"
+   width="14px"
+   inkscape:zoom="32"
+   inkscape:cx="-3.25"
+   inkscape:cy="7.375"
+   inkscape:window-width="1856"
+   inkscape:window-height="1011"
+   inkscape:window-x="64"
+   inkscape:window-y="32"
+   inkscape:window-maximized="1"
+   inkscape:current-layer="Warstwa_1"><sodipodi:guide
+     position="0,14"
+     orientation="0,14"
+     id="guide1041" /><sodipodi:guide
+     position="14,14"
+     orientation="14,0"
+     id="guide1043" /><sodipodi:guide
+     position="14,0"
+     orientation="0,-14"
+     id="guide1045" /><sodipodi:guide
+     position="0,0"
+     orientation="-14,0"
+     id="guide1047" /></sodipodi:namedview>
+
+<path
+   d="M 7.4991185,0 C 5.6649032,1.3333334 3.8342153,2.6666668 2.0000001,4.0000002 c 0,2.3315697 0,4.6666668 0,6.9982368 L 8.4973548,14.000001 14.000001,10 c 0,-2.3315694 0,-4.6666665 0,-6.9982362 v -0.00353 z M 12.571429,3.5132277 8.4973548,6.4726634 6.4338627,5.5449738 10.532628,2.5714287 Z M 7.4991185,1.1710759 9.4920639,2.0917108 5.3827163,5.0723106 3.4144622,4.186949 Z M 2.9982364,5.171076 4.9982365,6.0952383 5.0017639,8.9982367 6.0000002,9.4603178 V 6.5573195 L 8.0000003,7.4814818 V 12.59612 L 2.9982364,10.289242 Z M 13.001764,9.4920639 8.9982367,12.39859 V 7.347443 L 13.001764,4.4338626 Z"
+   id="path62"
+   style="stroke-width:0.0352734" />
```

Comment:
Icons [must use SVG fills only, not SVG strokes](https://github.com/gravitystorm/openstreetmap-carto/blob/master/CONTRIBUTING.md#map-icon-guidelines)